### DISCUSSION TITLE
General Set Up Issues

### DISCUSSION THREAD

**Discussion by ghosty999** [rodit/SnapMod#222] [Q&A/Support]
Hi there, I'm on a rooted/Magisk OnePlus 8T. I downloaded 11.82.0.32 version of Snapchat via the Arora store. I then downloaded the 1.8.3 of Snapmod APK and installed. 

It then shows the image below
![signal-2022-07-19-134918_002](https://user-images.githubusercontent.com/1349228/179754224-2882cff0-95f7-4b73-8835-f0905983ac9e.jpeg)

I download the file and install it, then the mappings are downloaded and applied. However the app says "Error getting Snapchat package info"

I reinstall both apps, I also force close each app twice, clear cache, restart the phone many times. I tried copied the version number .JSON into the files folder, I tried copying the mappings over. 

Am I being a n00b? (probably). Was there a .zip module I'm supposed to stall in Magisk? I've read the Wiki and Discussions many many times.

**Comment by authorisation** [comment]
Did you enable Snapmod in LSPosed?

**Reply by ghosty999** [reply]
> Did you enable Snapmod in LSPosed?

I didn't see any mention of this in the Wiki or ReadMe. Just installing now. Thanks!
